kamailio/users/config/kamailio.cfg

```diff
@@ -561,6 +562,38 @@ route[ADAPT_RURI_IN] {
     $rU = $var(transformated);
 }
 
+# Detects routing tag, sets header with tag and strips it from r-uri
+route[GET_ROUTING_TAG] {
+    if ($dlg_var(type) == 'wholesale') return;
+
+    # Guess routing tag
+    sql_query("cb", "SELECT tag FROM RoutingTags RT INNER JOIN CompaniesRelRoutingTags CRRT ON CRRT.routingTagId = RT.id WHERE CRRT.companyId = '$dlg_var(companyId)'", "rb");
```

Comment:
prefix column for RoutingTags is called `prefix` unless you're suggesting renaming to `tag` to match this query

Reply:
Sorry, I suggested the renaming to the mate on the right but I forgot telling you...

Reply:
Don't bother with that guy, he's as handsome as forgetful